roles/default/kiali-deploy/templates/kubernetes/role.yaml

```diff
@@ -50,10 +50,7 @@ rules:
   - patch
   - watch
 - apiGroups:
-  - config.istio.io
   - networking.istio.io
-  - authentication.istio.io
-  - rbac.istio.io
```

Comment:
I think you also want to do this to the openshift role.yaml template. This: https://github.com/kiali/kiali-operator/blob/master/roles/default/kiali-deploy/templates/openshift/role.yaml#L52-L57